src/utils.ts

```diff
@@ -110,7 +110,7 @@ export function getTranslationDirectoryKey(
  * @returns whether the response includes NAK
  */
 export function isNAK(response: string): boolean {
-    return response.includes("NAK");
+    return response.includes("NAK") && !response.includes("ACK");
```

Comment:
for my own understanding, why do we need to check !"ack"?

Reply:
Thanks for pointing this out, I added this to ensure we're not handling any message including both ACK and NAK as either ACK/NAK, and instead defaulting to an invalid response